src/condition.cpp

```diff
@@ -243,6 +243,30 @@ str_or_var get_str_or_var( const JsonValue &jv, const std::string &member, bool
     return ret_val;
 }
 
+translation_or_var get_translation_or_var( const JsonValue &jv, const std::string &member,
+        bool required, const translation &default_val )
+{
+    translation_or_var ret_val;
+    translation str_val;
+    if( jv.read( str_val ) ) {
+        ret_val.str_val = str_val;
+    } else if( jv.test_object() ) {
+        const JsonObject &jo = jv.get_object();
+        if( jo.has_member( "mutator" ) ) {
+            // if we have a mutator then process that here.
+            ret_val.function = conditional_t::get_get_string( jo );
```

Comment:
All the current mutators read `type_id`s or tripoints which are not translatable so string is fine.

Reply:
I think `translation_or_val` should not contain a `function` value then, which can be implemented later if needed, because if it's ever going to be used it should be translatable.

Reply:
It turns out that there are values of "mutator" that return translated strings, so I went ahead and split `get_get_string` into translatable and non-translatable components, so that `get_get_translation` can get the translation object from these cases.

Tested with:
```json
"u_query": { "mutator": "ma_technique_name", "matec_id": "WBLOCK_1" }
```
In French, it'll use "bloquer" in the query message.